### DISCUSSION TITLE
Include tautomers to substructure search

### DISCUSSION THREAD

**Discussion by ivannnnnnnnnn** [rdkit/rdkit#6245] [Development]
Hello, I am using rdkit 2022.9.3 and postgresql cartridge.  

When I make substructure search for this molecule I recieve many results in my database
SMILES: c1ccn2cnnc2c1
![image](https://user-images.githubusercontent.com/58053056/228559971-0fcfe854-1d96-44d8-a943-a5dbd56c4584.png)

But when I search by this molecule I dont receive any results
SMILES: N/N=c1/cccc[nH]1
![image](https://user-images.githubusercontent.com/58053056/228560366-ab358eb7-6bf2-4138-90d9-e37dce8483a6.png)

These molecules are tautomers and I would like to get the same results for the second molecule as I did for the first one. 

The search in ISIS base works like this

Can rdkit be configured so that the search works the same way?

Thank you

**Comment by greglandrum** [comment]
Hi @ivannnnnnnnnn the rdkit does support tautomer-insensitive substructure searching from Python (https://greglandrum.github.io/rdkit-blog/posts/2021-08-03-generalized-substructure-search.html), but this is not yet supported in the cartridge.

**Reply by ivannnnnnnnnn** [reply]
I tried this code, but this example not create all needed tautomers for molecules from example and don't solve my problem

**Comment by sitanshubhunia** [comment]
@greglandrum can we expect this feature can include in cartridge in near future..



Thanks in advance 🙏

**Comment by kienerj** [comment]
I'm not sure this is actually a tautomer issue and not an aromaticity issue? The nitrogens in the "side chain" won't match the "same" nitrogens in the ring if the ring is aromatic.

rdkit in that regard while technically correct can be confusing compared to "commercial cartridges"

**Comment by trallnag** [comment]
Feature request here: https://github.com/rdkit/rdkit/issues/6353
